### PR TITLE
Add usage information to python cleanup scripts

### PR DESCRIPTION
**Pull request by dpryan79**
Currently, the python scripts in `scripts/cleanup_datasets` don't provide any usage information (and will generally throw an error if run without any input files). This pull request endeavours to change that.

### CODE REVIEW

**Comment by bgruening** [comment]
LGTM! Very much appreciated.
@dpryan79 please also have a look at the [gravity project](https://github.com/galaxyproject/gravity)

File: scripts/cleanup_datasets/populate_uuid.py [hunk 1]
```diff
@@ -22,7 +22,19 @@
 
 assert sys.version_info[:2] >= ( 2, 4 )
 
+def usage(prog) :
+    print "usage: %s galaxy.ini" % prog
+    print """
+Populates blank uuid fields in datasets with randomly generated values
```
Comment (nsoranzo) [inline scripts/cleanup_datasets/populate_uuid.py]: A closing full stop is missing.

File: scripts/cleanup_datasets/remove_renamed_datasets_from_disk.py [hunk 2]
```diff
@@ -8,7 +8,21 @@
 
 assert sys.version_info[:2] >= ( 2, 4 )
 
+def usage(prog) :
+    print "usage: %s file" % prog
+    print """
+Removes a set of files from disk. The input file should contain a list of files
+to be deleted, one per line. The full path must be specified and must begin
+with /var/opt/galaxy.
```
Comment (nsoranzo) [inline scripts/cleanup_datasets/remove_renamed_datasets_from_disk.py:16]: This seems a pretty strong limitation. Is this script still useful?

File: scripts/cleanup_datasets/rename_purged_datasets.py [hunk 3]
```diff
@@ -8,7 +8,22 @@
 
 assert sys.version_info[:2] >= ( 2, 4 )
 
+def usage(prog) :
+    print "usage: %s file" % prog
+    print """
+Marks a set of files as purged and renames them. The input file should contain a
+list of files to be purged, one per line. The full path must be specified and
+must begin with /var/opt/galaxy.
```
Comment (nsoranzo) [inline scripts/cleanup_datasets/rename_purged_datasets.py:16]: As above, this seems a pretty strong limitation. Is this script still useful?

File: scripts/cleanup_datasets/populate_uuid.py (hunk 1 above)
Comment (dpryan79) [inline scripts/cleanup_datasets/populate_uuid.py]: I just pushed a commit to add the period.

File: scripts/cleanup_datasets/rename_purged_datasets.py (hunk 3 above)
Comment (dpryan79) [inline scripts/cleanup_datasets/rename_purged_datasets.py:16]: I would argue that they're probably not useful for most people. I have no clue why /var/opt/galaxy is hard-coded, though at least now people can know that without going through the code themselves. Personally I'd prefer if all of these scripts just parse the galaxy.ini for things like this, but that's probably a different PR.

**Comment by dannon** [comment]
+1, thanks for this @dpryan79!

**Pull request merged**